.npmrc

```diff
@@ -1,10 +1,11 @@
 # This file defines default variables values used by npm scripts.
 # Read more at at:  https://docs.npmjs.com/files/npmrc
 # Config variables can be overridden by running:
-# npm run <script-name> --kubernetes-dashboard:<variable-name>=<path>
+# npm run <script-name> --kubernetes-dashboard:<variable-name>=<value>
 
 kubernetes-dashboard:kubeconfig_dir = ${HOME}/.kube
 kubernetes-dashboard:kubeconfig = ${HOME}/.kube/config
 kubernetes-dashboard:frontend_build_dir = dist/frontend/en
 kubernetes-dashboard:dashboard_binary_path = dist/amd64/dashboard
 kubernetes-dashboard:dashboard_locale_config = dist/amd64/locale_conf.json
+kubernetes-dashboard:insecure_bind_address = 0.0.0.0
```

Comment:
Use localhost as default bind address here. Binding to all interfaces by default is not secure.